pipeline/outputs/elasticsearch.md

```diff
@@ -115,6 +120,45 @@ In your main configuration file append the following _Input_ & _Output_ sections
 
 ![example configuration visualization from calyptia](../../.gitbook/assets/image%20%282%29.png)
 
+### Configuration File with Upstream
+
+In your main configuration file append the following _Input_ & _Output_ sections.
+
+```text
+[INPUT]
+    Name     cpu
+    Tag      cpu
+
+[OUTPUT]
+    Name     es
+    Match    *
+    Upstream ./upstream.conf
+    Index    my_index
+    Type     my_type
+```
+
+Your [Upstream Servers](../../administration/configuring-fluent-bit/classic-mode/upstream-servers.md) configuration file can look like:
```

Comment:
```suggestion
Your [Upstream Servers](../../administration/configuring-fluent-bit/classic-mode/upstream-servers.md)
configuration file can similar to the following:
```

Reply:
Suggestion was applied as part of f28a9c6a402fcf5dc87b1aed0902e1dff09c6749.